src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js

```diff
@@ -56,6 +57,14 @@ export const BulkEditInApp = (
             getFilteredFields={getFilteredFields}
           />
         )}
+
+        {typeOfBulk === CAPABILITIES.HOLDINGS && (
+          <HoldingsForm
+            onContentUpdatesChanged={onContentUpdatesChanged}
+            typeOfBulk={typeOfBulk}
```

Comment:
Is `typeOfBulk` used in `HoldingsForm` ?

Reply:
Ye, It's an unused prop. I will remove it